posthog/models/plugin.py

```diff
@@ -570,7 +574,33 @@ def preinstall_plugins_for_new_organization(sender, instance: Organization, crea
 
 @receiver(models.signals.post_save, sender=Team)
 def enable_preinstalled_plugins_for_new_team(sender, instance: Team, created: bool, **kwargs):
-    if created and can_configure_plugins(instance.organization):
+    if not created or not can_configure_plugins(instance.organization):
+        return
+
+    if getattr(settings, "HOG_TRANSFORMATIONS_ENABLED", True):
```

Comment:
Default to true is almost never a good idea. Also the default should happen in the settings.